<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>module3</class>
 <widget class="QWidget" name="module3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page0">
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: transparent;
border-image: url(:/modules/m3p1.png);
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="page1">
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: transparent;
border-image: url(:/modules/m3p2.png);
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="page2">
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: transparent;
border-image: url(:/modules/m3p3.png);
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="page3">
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: transparent;
border-image: url(:/modules/m3p4.png);
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="page4">
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: transparent;
border-image: url(:/modules/m3p5.png);
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="backModuleS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#backModuleS{
image: url(:/buttons/buttons/left.png);
background: transparent;
}
#backModuleS:hover{
	image: url(:/buttons/buttons/leftH.png);
}
#backModuleS:pressed{
	image: url(:/buttons/buttons/leftP.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPart">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#nextPart{
image: url(:/buttons/buttons/test.png);
background: transparent;
}
#nextPart:hover{
	image: url(:/buttons/buttons/testH.png);
}
#nextPart:pressed{
	image: url(:/buttons/buttons/testP.png);
}
#nextPart:disabled{
	image: url(:/buttons/buttons/testD.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="partText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>850</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>nextPart</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="anterior">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>500</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/buttons/buttons/left.png);
background: transparent;
}
QPushButton:hover{
	image: url(:/buttons/buttons/leftH.png);
}
QPushButton:pressed{
	image: url(:/buttons/buttons/leftP.png);
}
QPushButton:disabled{
	image: url(:/buttons/buttons/leftD.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="siguiente">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>500</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/buttons/buttons/right.png);
background: transparent;
}
QPushButton:hover{
	image: url(:/buttons/buttons/rightH.png);
}
QPushButton:pressed{
	image: url(:/buttons/buttons/rightP.png);
}
QPushButton:disabled{
	image: url(:/buttons/buttons/rightD.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
